CHANGELOG.md

```diff
@@ -17,7 +17,8 @@ version directory, and  then changes are introduced.
 
 ### Changed
 
-- Mount relevant directories so that the command `docker` can run in `Kubelet`. This is needed for `rbd` to mount `Ceph` volumes on the nodes.
+- Mount relevant directories so that the command `docker` can run in `Kubelet`. This is needed for `rbd` to mount `Ceph` volumes on the nodes.h
```

Comment:
My knowledge is shallow here. What is `rbd`?
```suggestion
- Mount relevant directories so that the command `docker` can run in `Kubelet`. This is needed for `rbd` to mount `Ceph` volumes on the nodes.
```